starfish/intensity_table/intensity_table.py

```diff
@@ -389,10 +394,29 @@ def from_image_stack(
         return IntensityTable.from_spot_data(intensity_data, pixel_coordinates)
 
     @staticmethod
-    def concatanate_intensity_tables(intensity_tables: List["IntensityTable"]):
-        # TODO VARY CONCAT LOGIC IF TILES OVERLAP
-        # This method is a starting point for handling tile overlap, right now
-        # it does a simple concat but people want other overlap logic implmented
+    def process_overlaps(its: List["IntensityTable"],
+                         overlap_strategy: OverlapStrategy
+                         ) -> List["IntensityTable"]:
+        """Find the overlapping sections between IntensityTables
+        and process them according to the give overlap strategy
```

Comment:
```suggestion
        and process them according to the given overlap strategy
```

you can probably unwrap this a bit as well.